package.json

```diff
@@ -8,12 +8,15 @@
   "scripts": {
     "pretest": "lerna bootstrap",
     "prepublish": "cp readme.md packages/polka/.",
-    "test": "tape tests/*.js | tap-spec"
+    "test": "tape tests/*.js | tap-spec",
+    "lint": "biome lint bench/*.js && biome lint packages/**/*.js",
+    "lint:fix": "biome lint bench/*.js --apply-unsafe && biome lint packages/**/*.js --apply-unsafe"
   },
   "devDependencies": {
-    "axios": "^0.19.0",
-    "lerna": "^2.8.0",
-    "tap-spec": "^4.1.1",
-    "tape": "^4.8.0"
+    "@biomejs/biome": "1.5.3",
+    "axios": "0.19.0",
```

Comment:
why pin dependencies everywhere?

Reply:
it's not a good practice because it prevents dependency deduping. package managers all have lockfiles to make sure you use an exact version

Reply:
I pin versions. It's tooling for the repo and should be locked down for sanity and reproducible builds. Dependencies should always be ranged.